.github/CONTRIBUTING.md

```diff
@@ -209,15 +208,15 @@ format that includes a **type** and a **subject**:
 
 This is the list of _type_ of commits that we accept:
 
--   ast
--   break
--   chore
--   cli
--   docs
--   feat
--   fix
--   misc
--   tests
+-   **ast** : init, migrate, etc.
```

Comment:
```suggestion
-   **ast** : Init, migrate, etc.
```

Reply:
This message doesn't give enough context, need an alternative

Reply:
Something you have in mind??

Reply:
cc /@webpack/cli-team